apps/nextjs-website/src/lib/webinars/webinarQuestions.ts

```diff
@@ -35,64 +33,50 @@ export type WebinarQuestion = {
   readonly highlightedBy?: string;
 };
 
+export type UpdateExpression<T> =
+  | {
+      readonly operation: 'update';
+      readonly value: T;
+    }
+  | {
+      readonly operation: 'remove';
+    };
+
+export type WebinarQuestionUpdate = Pick<
+  WebinarQuestion,
+  'webinarId' | 'createdAt'
+> & {
+  readonly hiddenBy?: UpdateExpression<string>;
+  readonly highlightedBy?: UpdateExpression<string>;
+};
+
 export const insertWebinarQuestion = (question: InsertWebinarQuestion) =>
   pipe(
     R.ask<Pick<WebinarEnv, 'dynamoDBClient' | 'nowDate'>>(),
```

Comment:
This might not be clear from people with not a so strong fp background that is reading values from WebinarEnv, i'd add a comment explaining this line to make it more readable